src/components/pdf-upload/PdfList.tsx

```diff
@@ -247,19 +392,57 @@ export function PdfList({ refreshTrigger }: PdfListProps) {
               </div>
             ) : (
               <>
+                {/* Show which campaigns the file is already in */}
+                {selectedFile?.campaigns &&
+                  selectedFile.campaigns.length > 0 && (
+                    <div className="mb-4 p-3 bg-purple-50 dark:bg-purple-900/10 rounded-md">
+                      <p className="text-sm font-medium text-purple-800 dark:text-purple-300 mb-2">
+                        Linked campaigns:
+                      </p>
+                      <div className="flex flex-wrap gap-1">
+                        {selectedFile.campaigns.map((campaign) => (
+                          <span
+                            key={campaign.campaignId}
+                            className="px-2 py-1 text-xs bg-purple-100 dark:bg-purple-900/20 text-purple-800 dark:text-purple-300 rounded"
+                          >
+                            {campaign.name}
+                          </span>
+                        ))}
+                      </div>
+                    </div>
+                  )}
+
                 <fieldset className="mb-4">
                   <legend className="block text-sm font-medium text-gray-700 dark:text-gray-300 mb-2">
                     Select campaigns
                   </legend>
-                  <MultiSelect
-                    options={campaigns.map((campaign) => ({
-                      value: campaign.campaignId,
-                      label: campaign.name,
-                    }))}
-                    selectedValues={selectedCampaigns}
-                    onSelectionChange={setSelectedCampaigns}
-                    placeholder="Choose campaigns..."
-                  />
+                  {campaigns.filter(
```

Comment:
Do we need to do this filter if the resource<->campaign logic has already been filtered above (~L80)?